virtcontainers/container.go

```diff
@@ -873,8 +874,11 @@ func (c *Container) create() (err error) {
 	}
 	c.process = *process
 
-	if err = c.newCgroups(); err != nil {
-		return
+	// cgroups are not yet supported with rootless execution
```

Comment:
So is this a system/kernel limitation?

Reply:
I think it depends on if kata supports [cgroups v2](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=34a9304a96d6351c2d35dcdc9293258378fc0bd8), and if so would need to work with podman to get that working. It looks like they have an issue open on getting that working with runc:
https://github.com/containers/libpod/issues/1429

Reply:
Interesting change.  Does podman manage any cgroups that we execute in?

/cc @jcvenegas